target/linux/generic/config-6.12

```diff
@@ -5161,6 +5252,8 @@ CONFIG_RTC_SYSTOHC_DEVICE="rtc0"
 CONFIG_RT_MUTEXES=y
 CONFIG_RUNTIME_TESTING_MENU=y
 # CONFIG_RUST is not set
+CONFIG_RUSTC_LLVM_VERSION=0
+CONFIG_RUSTC_VERSION=0
```

Comment:
I think these two symbols should be filtered by `config-filter`.

Reply:
`# CONFIG_RUSTIC_.* is not set` ?

Reply:
It's better to use full symbol names.
```
CONFIG_RUSTC_LLVM_VERSION=.*
CONFIG_RUSTC_VERSION=.*
```

Reply:
Please analyse: https://github.com/namiltd/openwrt/pull/35/commits/d85e285b1eadd83c465227954760a3d227f90618